app/person.py

```diff
@@ -0,0 +1,56 @@
+
+""" Amity models"""
+
+
+class Person(object):
+    """docstring for Person."""
+
+    def __init__(self, first_name, last_name, gender):
+        self.first_name = first_name
+        self.last_name = last_name
+        self.gender = gender
+
+    def __str__(self):
+        return '{} {}'.format(self.first_name, self.last_name)
+
+
+class Fellow(Person):
+    """docstring for Fellow."""
```

Comment:
This is not how you write a function doc string. Please put a better description than what is there.